### PR TITLE
feat: automate release

### PR DESCRIPTION
**Pull request by TrickyPi**
related #136

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        run: npm ci
+        env:
+          PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: "1"
+
+      - name: build
+        run: |
+          npm run build
+
+      - name: publish to npm
+        run: npm publish
```
Comment (jpillora) [inline .github/workflows/release.yml:36]: What do we think about setting version to “0.0.0-git-tag” in main?
And then sed replacing it to git ref name (tag) before publish?

Comment (jpillora) [reply .github/workflows/release.yml:36]: do we need to commit to main?

would this work:

`main` will permanent have a package.json with “0.0.0-git-tag”
`gh-pages` will be main + build files + package.json with git ref name (tag) - and this is what we publish?

Comment (TrickyPi) [reply .github/workflows/release.yml:36]: like this:
image

it's weird.

Comment (jpillora) [reply .github/workflows/release.yml:36]: oh yea, server needs to be just `1.5.0` (no `v`) 

and the `0.0.0-git-tag` is supposed to mean "not versioned here, check the git tag"

so we wouldn't have a `0.0.0` tag

once this is working, we'll tag `1.5.0` and it'll publish that

Comment (TrickyPi) [reply .github/workflows/release.yml:36]: Oh, the `0.0.0` tag in the picture is just for illustration , not a real tag.

Comment (TrickyPi) [reply .github/workflows/release.yml:36]: > `main` will permanent have a package.json with “0.0.0-git-tag”
> `gh-pages` will be main + build files + package.json with git ref name (tag) - and this is what we publish?

Hmm, make senses.

Comment (jpillora) [reply .github/workflows/release.yml:36]: are you happy to make these changes? im also okay to merge this PR, and then i can work on the changes, just not 100% when ill get time

Comment (TrickyPi) [reply .github/workflows/release.yml:36]: I will make these changes later.😄

Comment (TrickyPi) [reply .github/workflows/release.yml:42]: Done

**Pull request merged**